region/crateria/east.json

```diff
@@ -1833,6 +1878,43 @@
               ]
             }
           ]
+        },
+        {
+          "from": 17,
+          "to": [
+            {
+              "id": 10,
+              "strats": [
+                {
+                  "name": "West Ocean G-mode Overload PLMs by PBing Missile Item",
+                  "notable": true,
+                  "requires": [
+                    {"itemNotCollectedAtNode": 10},
+                    "h_canUsePowerBombs",
+                    {"or": [
+                      "h_canFly",
+                      {"and": [
+                        "canTrickyDashJump",
+                        "canSpringBallJumpMidAir",
+                        "HiJump"
+                      ]}
+                    ]}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "B",
+                      "requires": []
+                    }
+                  ],
+                  "note": [
+                    "To overload the PLMs, place a PB precisely to the right of the bottom of the second overhang above the door to the Moat.",
+                    "This is at the max jump height without HiJump. Placing the PB higher or lower will not overload the PLMs without many PBs."
```

Comment:
This should work by placing the PB in range of most of the bomb blocks.

Reply:
I worked on this a lot yesterday. Placing the PB in this precise location works with 1PB. placing it 1 tile lower takes ~15 PBs and placing it 1-2 tiles higher also takes ~15 PBs.

Reply:
This was just a speculation comment, I didn't mean to act like I knew how it worked.